### PR TITLE
Flow C1 upgrade implementation

### PR DESCRIPTION
**Pull request by sone-md**
We have completed the upgrade to support Flow Cadence 1.0. Please check and confirm.

### CODE REVIEW

File: transactions/FanTop/initialize_collection.cdc [hunk 1]
```diff
@@ -1,16 +1,16 @@
-import FanTopToken from 0xFANTOP_ADDRESS
+- Arguments: []
```
Comment (scottphc) [inline transactions/FanTop/initialize_collection.cdc:1]: please remove `Arguments` and keep the original script.

Comment (sone-md) [reply transactions/FanTop/initialize_collection.cdc]: thanks, I fixed it.

File: transactions/FanTop/initialize_collection.cdc [hunk 2]
```diff
@@ -1,16 +1,14 @@
-import FanTopToken from 0xFANTOP_ADDRESS
+import FanTopToken from 0x86185fba578bc773
```
Comment (scottphc) [inline transactions/FanTop/initialize_collection.cdc:1]:
```suggestion
import FanTopToken from 0xFANTOP_ADDRESS
```

File: transactions/FanTop/sell_token.cdc [hunk 3]
```diff
@@ -1,5 +1,6 @@
-import FanTopToken from 0xFANTOP_ADDRESS
-import FanTopPermissionV2a from 0xFANTOP_ADDRESS
+import NonFungibleToken from 0x1d7e57aa55817448
+import FanTopToken from 0x86185fba578bc773
+import FanTopPermissionV2a from 0x86185fba578bc773
```
Comment (scottphc) [inline transactions/FanTop/sell_token.cdc:3]:
```suggestion
import NonFungibleToken from 0xNON_FUNGIBLE_TOKEN_ADDRESS
import FanTopToken from 0xFANTOP_ADDRESS
import FanTopPermissionV2a from 0xFANTOP_ADDRESS
```

File: transactions/FanTop/transfer_token.cdc [hunk 4]
```diff
@@ -1,15 +1,16 @@
-import FanTopToken from 0xFANTOP_ADDRESS
+import NonFungibleToken from 0x1d7e57aa55817448
+import FanTopToken from 0x86185fba578bc773
```
Comment (scottphc) [inline transactions/FanTop/transfer_token.cdc:2]:
```suggestion
import NonFungibleToken from 0xNON_FUNGIBLE_TOKEN_ADDRESS
import FanTopToken from 0xFANTOP_ADDRESS
```

**Review by scottphc — CHANGES_REQUESTED** [review]

File: transactions/FanTop/transfer_token.cdc (hunk 4 above)
Comment (sone-md) [reply transactions/FanTop/transfer_token.cdc]: Thank you for your support!

**Review by scottphc — APPROVED** [review]

**Comment by sone-md** [comment]
@scottphc 
Thank you for confirming and merged. Is there a scheduled release for this version?

**Comment by scottphc** [comment]
> @scottphc Thank you for confirming and merged. Is there a scheduled release for this version?

released already

**Pull request merged**